### DISCUSSION TITLE
Homes, First interaction only with Bed (Sleep/Click/place) sets first home.

### DISCUSSION THREAD

**Discussion by TomLewis** [Zrips/CMI#7293] [Enhancement]
Theres a bed feature to set a home when interacted with, but this is always on, it would be much better to allow only first time use of a bed to set a home! and run a command so that we can send a message to people how to use /homes and how to set new homes!

It would be great if we could trigger it for placing your first bed, so that they get sent a message in chat (via a command) and then it can also set their first home automatically!

This feature is just annoying to long term players that understand how to use homes!

**Comment by mrfloris** [comment]
In regard to /homes, .. you could go to alias.yml and set enabled: false, and then use /cmi aliaseditor to add a new command called homes, then using specialized commands you can check how many homes someone has, if they don't have any, give them a msg on how to set homes. And if they have 1 or more, then run /cmi homes for the player. 

The right click on bed is one home, using /sethome, will set another Home, both will be called Home, unless you customize it in the configuration yml file under settings/

I indeed can't think of an existing way for this edge-case where you want them to only use a bed as a first home, and have that be the same as /sethome Home. (just to let you know).

**Comment by TomLewis** [comment]
Yeah, ive been trying to wrack my brains to think how to do it, it definitally needs to be something that happens without them reading/learning. Currently we use GP which when they place their first check the area is auto claimed, so auto placing their first bed would be PERFECT way to solve them not learning that /homes exists.

The way I found this out, Is I recently had to write a script to convert Essentails homes to CMI, as for years and years ive never been able to process Ess to CMI homes as Ive had too many users, and it just crashes my server every time Ive tried, so I gave up and just wrote a script myself to do this.

Now that all the homes are in the CMI database, i was able to run some SQL queries and was shocked at how little people had set a home!
